components/console-backend-service/before-commit.sh

```diff
@@ -93,14 +93,14 @@ if [[ ${buildErrCheckResult} != 0 ]]; then
     exit 1
 fi
 
-errCheckResult=$(./errcheck-vendored -blank -asserts -ignoregenerated ./...)
-rm errcheck-vendored
-
-if [[ $(echo ${#errCheckResult}) != 0 ]]; then
-    echo -e "${RED}✗ [errcheck] unchecked error in:${NC}\n${errCheckResult}${NC}"
-    exit 1
-else echo -e "${GREEN}√ errcheck ${NC}"
-fi
+#errCheckResult=$(./errcheck-vendored -blank -asserts -ignoregenerated ./...)
+#rm errcheck-vendored
+#
+#if [[ $(echo ${#errCheckResult}) != 0 ]]; then
+#    echo -e "${RED}✗ [errcheck] unchecked error in:${NC}\n${errCheckResult}${NC}"
+#    exit 1
+#else echo -e "${GREEN}√ errcheck ${NC}"
+#fi
```

Comment:
decide what shoudl be done here - uncomment or remove it

Reply:
autogenerated files need this line at the beginning: `// Code generated by mockery v1.0.0. DO NOT EDIT.` to skip them in this check... uncommented